blockade/tests/test_integration_rest.py

```diff
@@ -178,3 +178,60 @@ def test_network_state_slow(self):
         result = requests.post(url, headers=self.headers, data=data)
         assert result.status_code == 204
         self._assert_container_network_state('c1', 'SLOW')
+
+    def _test_basic_events(self, event):
+        data = '''
+            {
+                "event_set": ["%s"],
+                "min_start_delay": 1,
+                "max_start_delay": 2,
+                "min_run_time": 30000,
+                "min_run_time": 300000,
+                "min_containers_at_once": 2,
+                "max_containers_at_once": 2
+            }
+        ''' % event
+        url = self.url + '/chaos'
+        result = requests.post(url, headers=self.headers, data=data)
+        assert result.status_code == 201
+        time.sleep(10.0)
```

Comment:
I need to make sure that chaos has a chance to run.  It should be scheduled after a 1ms delay.  Once that delay expires it will start the blockade event.  However that blockade event must complete before the state will change.  Those operations tend to take a bit of time.  This is sadly not a fully deterministic test.